src/prototype_room_my.js

```diff
@@ -128,13 +128,14 @@ Room.prototype.handleObserver = function() {
 
     // TODO scan full range, first implementation
     const nameSplit = this.splitRoomName();
-    const fullLength = 2 * OBSERVER_RANGE + 1;
+    const observerRange = Math.max(Math.min(config.room.observerRange, OBSERVER_RANGE), 1);
```

Comment:
Why 1 instead of 0? Is there another way to disable observer scanning, or would a user want to set config.room.observerRange to zero to disable their scanning?

Reply:
True, this should be `0`

Reducing to 1 commit is appreciated, too